BREAKING-CHANGES.md

```diff
@@ -40,6 +41,29 @@ This is a quick summary of the sections below:
 
 ## Unreleased (v0.24.1)
 
+### `List::start_corner` is renamed to `List::direction` ([#673])
+
+[#673]: https://github.com/ratatui-org/ratatui/pull/673
+
+Previously `List::start_corner` didn't communicate the intent of the method. It also used an
+inadequate `Corner` enum. The method is now renamed `direction` and a new enum
+`ListDirection` has been added.
+
+```rust
+List::new(/* items */).start_corner(Corner::TopLeft);
+List::new(/* items */).start_corner(Corner::TopRight);
+// This is not an error, BottomRight rendered top to bottom previously
+List::new(/* items */).start_corner(Corner::BottomRight);
+// all becomes
+List::new(/* items */).direction(ListDirection::TopToBottom);
+```
+
+```rust
+List::new(/* items */).start_corner(Corner::BottomLeft);
+// becomes
+List::new(/* items */).direction(ListDirection::BottomToTop);
+```
```

Comment:
Just an idea, we can probably use a diff here like below to highlight the changes:

```diff
- List::new(/* items */).start_corner(Corner::BottomLeft);
+ List::new(/* items */).direction(ListDirection::BottomToTop);
```

What do you think?

Reply:
That's a good idea, I'll add it when that makes sense